docs/book/developer-guide/advanced-concepts/step-fixtures.md

```diff
@@ -0,0 +1,255 @@
+---
+description: How to use step fixtures to access the active ZenML stack from within a step.
+---
+
+# Step Fixtures
```

Comment:
Makes sense, I think this should be done for all advanced concepts. Will rename advanced concepts to "Advanced Usage" or similar and then rename all files to highlight what they do instead of how we call the internal concept.

Reply:
ok ill leave this unresolved for now